### ISSUE TITLE
Please specify the filter widget

### ISSUE THREAD

**Issue by Chealer** [Mottie/tablesorter#1321]
The filter widget seems very useful. It has several example pages, but I cannot find a page with real documentation.

I am specifically looking for a way to add filters for some columns, but not all.

**Comment by Mottie** [comment]
Hi @Chealer!

The [filter widget *options*](https://mottie.github.io/tablesorter/docs/#Widget-options) are included on detail on the main documentation page. More details about specific class names, methods, events and how to modify localization are found on the main ["filter widget" page](https://mottie.github.io/tablesorter/docs/example-widget-filter.html). Because of duplication, the filter widget page options are not as detailed as the documentation on the main documentation page.

To answer your question, you can disable filters for specific columns by adding a "filter-false" class to the column ([ref](https://mottie.github.io/tablesorter/docs/example-widget-filter.html#classes)). This will still add a filter, but it will be disabled. To completely hide the filter, you'll need to include some extra css:

```css
.tablesorter-filter.disabled {
  display: none;
}
```

For future reference, please don't open issues to ask a question. You can use Gitter, IRC or stackoverflow -  the links can be found in the [readme](https://github.com/Mottie/tablesorter#questions) on the main repo page.

**Comment by Chealer** [comment]
Thank you @Mottie. This was not a question, I was giving an example of something which might be the result of insufficient documentation, but I see this case is more a case of missing feature. Thanks for the workaround suggestion.

Another specific information which seemed to miss is that the HTML attribute data-column is used to configure filtering. I now see that this is documented in the [basic page](https://mottie.github.io/tablesorter/docs/example-widget-filter.html), but only in the Methods section, which is collapsed by default, and which is not where the reader would expect to find DOM information. One more thing is that this page's URL presents it as an example. One would not expect to find specifications in examples.

**Comment by Mottie** [comment]
I would appreciate any help you could provide with the documentation. I know it's a bit fragmented, but I think just about everything *is* documented. Maybe there should be a FAQ above the notes section?

**Comment by Chealer** [comment]
I agree the quantity is present. I think the structure is more problematic, indeed. The documentation of PHP extensions, such as [the SDO extension's documentation](http://php.net/manual/en/book.sdo.php), could be used, not necessarily as a model, but as inspiration. It starts with an introduction, then gives examples, and then formal specifications.

I believe the documentation index could link to an introduction page, which would then link to the example pages.

Regarding the notes section, I find it too prominent. Of course, I am a new user, so I am biased, and it is very important to document API changes, but still, the current state of a widget should be more prominent than its history. So I would certainly put an introduction/FAQ about the notes section.
